doc-Managing_Providers/topics/Adding_Amazon_EC2_Providers.adoc

```diff
@@ -10,6 +10,13 @@ Complete the following procedure to add an Amazon EC2 cloud provider in {product
 . Select a *Region*.
 . Select the appropriate *Zone* if you have more than one available.
 . Under *Endpoints*, click the *Default* tab.
+.. Enter the *Endpoint URL*. 
++
+[NOTE]
```

Comment:
Hey Suyog,

I would recommend restructuring the language in this note. In my opinion you can remove most of the first sentence, and have the note start with "AWS allows users to set a custom endpoint URL when connecting to certain services, which you can add in the {product-title_short} user interface per Amazon EC2 provider. see link:https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/vpce-interface.html[Interface VPC Endpoints (AWS PrivateLink)] for more information." 

I think that says pretty much everything in removes the need for the introductory sentence in this paragraph. Let me know what you think.

-Chris

Reply:
@cbudz Hi Chris, thanks for your review and suggestion - good point. I've modified the note in my latest commit. Thanks, Suyog